tools/molecule.spec

```diff
@@ -0,0 +1,53 @@
+%global srcname molecule
+%global pypiname molecule
+%global pkgname python3-%{srcname}
+%global forgeurl https://github.com/ansible-community/%{srcname}
+
+Name:    %{pkgname}
+Version: 0.1.0
+Release: 1%{?dist}
+Summary: ...
+%forgemeta
+URL:       %{forgeurl}
+Source:    %{forgesource}
+License:   BSD
+BuildArch: noarch
+
+BuildRequires: python3-devel
+BuildRequires: python3-wheel
+BuildRequires: python3-tox-current-env
```

Comment:
Neither wheel nor tox-current-env should be explicitly BuildRequired: They are both generated with `%pyproject_buildrequires -t`.